src/SerilogWeb.Classic.Mvc/SerilogWeb.Classic.Mvc.csproj

```diff
@@ -41,31 +41,31 @@
       <HintPath>..\..\packages\Microsoft.Web.Infrastructure.1.0.0.0\lib\net40\Microsoft.Web.Infrastructure.dll</HintPath>
     </Reference>
     <Reference Include="Serilog, Version=2.0.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Serilog.2.7.1\lib\net45\Serilog.dll</HintPath>
+      <HintPath>..\..\packages\Serilog.2.9.0\lib\net45\Serilog.dll</HintPath>
```

Comment:
I'd rather not bump this dependency here, as it will force library clients to update, even though it's not necessary. 

Also `SerilogWeb.Classic` itself currently uses v2.7.1 of Serilog.

(FYI, this has to be seen as a "minimum version of Serilog needed", and given we don't do anything fancy with Serilog, we are fine with an older version)

Reply:
Make sense. I will revert the change.